src/index.coffee

```diff
@@ -123,9 +124,9 @@ class ConnectApp
       steps.unshift liveReload(@livereload)
     if typeof @root == "object"
       @root.forEach (path) ->
-        steps.push connect.static(path)
+        steps.push connect.static(path, {index: if @index is true then "index.html" else @index})
     else
-      steps.push connect.static(@root)
+      steps.push connect.static(@root, {index: if @index is true then "index.html" else @index})
```

Comment:
Make an options variable above the `if` then pass that into both of these functions since they are identical.